packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -13,7 +15,16 @@ export type UseCustomValidityOptions = {
   /**
    * Component props as defined with `const props = defineProps()`
    */
-  props: CustomValidityProp & { modelValue?: unknown };
+  props: CustomValidityProp & {
+    modelValue?: unknown;
+    type?: (typeof TRANSLATED_INPUT_TYPES)[number] | string;
+    pattern?: string | RegExp;
+    maxlength?: number;
+    minlength?: number;
+    min?: number;
+    max?: number;
+    step?: number;
```

Comment:
We don't have this currently, right because thats only for number inputs?

Reply:
correct. the question is, will we remember to include it when building the number input if I remove it now?

Reply:
if we already have a ticket for it I can move the code to the ticket. what do you think?

Reply:
Sounds good

Reply:
done https://github.com/SchwarzIT/onyx/issues/1273